trunk/NDHMS/Routing/Reservoirs/Persistence_Level_Pool_Hybrid/module_persistence_levelpool_hybrid_state.F

```diff
@@ -0,0 +1,76 @@
+! This module defines and instantiates objects
+! for a hybrid type reservoir's state.
+! State holds and tracks dynamic/changing variables
+! that are only relevant to the given hybrid
+! reservoir object and not other modules or areas
+! of the system.
+
+#define seconds_in_day 86400
```

Comment:
+1

Reply:
@jdmattern-noaa fixing. Changing parameter attribute in the module.